test/duct/logger/timbre_test.clj

```diff
@@ -78,7 +79,30 @@
       (logger/log logger :report ::testing)
       (is (re-matches
            #"(?x)
-             \d\d-\d\d-\d\d\ \d\d:\d\d:\d\d\ [^\s]+\ 
-             REPORT\ \[duct\.logger\.timbre-test:78\]\ -\ 
+             \d\d-\d\d-\d\d\ \d\d:\d\d:\d\d\ [^\s]+\
+             REPORT\ \[duct\.logger\.timbre-test:78\]\ -\
              :duct\.logger\.timbre-test/testing\n"
            (slurp tempfile))))))
+
+(deftest restore-root-timbre-config-test
+  (testing "manages taoensso.timbre/*config* binding"
+    (let [init-binding tao/*config*
+          config {::logger/timbre {:level :info, :appenders {}}}]
+
+      (assert (not= init-binding config)
+              "This test assumes these values don't start with the same value.")
+
+      (try
+        (let [system (ig/init config)]
+
+          (is (= (::logger/timbre config) tao/*config*)
+              "root timbre config is set")
+
+          (ig/halt! system)
+
+          (is (= init-binding tao/*config*)
+              "root timbre config is restored"))
+
+        (finally
+          ;; ensure root binding restored
+          (tao/set-config! init-binding))))))
```

Comment:
It would be better to write this with less whitespace etc.:

```clojure
(try
  (let [system (ig/init config)]
    (is (= (::logger/timbre config) tao/*config*))
    (ig/halt! system)
    (is (= init-binding tao/*config*)))
  (finally
    (tao/set-config! init-binding)))))
```

Just as clear, in a fraction the space.